### PR TITLE
Foundation post

### PR DESCRIPTION
**Pull request by utstikkar**
Before being merging we need:
- to add a picture (Ben)
- check that donation button is displayed as intended in the post
- the author is indeed me (for some reason it was showing Aleks earlier...)
- proofread and modify the text as you please

### CODE REVIEW

File: _posts/2013-05-07-opentechschool-is-now-a-foundation.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "OpenTechSchool is now a Foundation!"
+description: ""
+category: 
+tags: ['opentechschool', 'foundation', 'announcement', 'membership', 'donation']
+author: amelie
+image: "TOADD"
```
Comment (gnunicorn) [inline _posts/2013-05-07-opentechschool-is-now-a-foundation.md:8]: hmm. Sadly, I don't have any group picture from that day. Any other ideas?

Comment (utstikkar) [inline _posts/2013-05-07-opentechschool-is-now-a-foundation.md:8]: Maybe we could simply use a big OTS logo as pic for that post?

Comment (stefanhoth) [inline _posts/2013-05-07-opentechschool-is-now-a-foundation.md:8]: How about this one: https://www.dropbox.com/s/x8g8v06pfdihhbq/PhotoGrid_1367930717942.png

Comment (gnunicorn) [inline _posts/2013-05-07-opentechschool-is-now-a-foundation.md:8]: Nice. Can we have this in widescreen? That is better for blog-articles ...

Comment (stefanhoth) [inline _posts/2013-05-07-opentechschool-is-now-a-foundation.md:8]: Ok, here in widescreen: https://www.dropbox.com/s/eyx03fs23ovzb22/PhotoGrid_1367931217840.png

**Pull request merged**